.gitignore

```diff
@@ -8,3 +8,11 @@ __pycache__
 notebooks/.ipynb_checkpoints
 _attic/
 *.pyc
+
+venv/
```

Comment:
Have you created a traditional `virtualenv`. I think that rather using `poetry` as in Readme is safer.